### ISSUE TITLE
LateInitializationError: Field '_seriesType@114283885' has not been initialized.

### ISSUE THREAD

**Issue by jordandevogelaere** [syncfusion/flutter-widgets#195]
Hi,

When I try to render my DoughnutChart I'm getting this error in my debug console. 

<img width="1213" alt="image" src="https://user-images.githubusercontent.com/16119444/115015003-56f28100-9eb3-11eb-9c0b-9c4ba7ec7647.png">

The chart is rendering on the screen but I'm figuring out why I'm get this error? 

```class ChartSampleData {
  final String x;
  final double y;
  final String text;

  ChartSampleData({this.x, this.y, this.text});
}

class GenderChart extends StatefulWidget {
  @override
  _GenderChartState createState() => _GenderChartState();
}

class _GenderChartState extends State<GenderChart> {
  final List<ChartSampleData> chartData = <ChartSampleData>[
    ChartSampleData(x: 'Men', y: 55, text: '55%'),
    ChartSampleData(x: 'Woman', y: 31, text: '45%'),
  ];

  @override
  Widget build(BuildContext context) {
    return _buildDefaultDoughnutChart();
  }

  /// Return the circular chart with default doughnut series.
  SfCircularChart _buildDefaultDoughnutChart() {
    return SfCircularChart(
      title: ChartTitle(
          text: 'Composition of gender',
          textStyle: Get.theme.textTheme.headline6
              .copyWith(fontWeight: FontWeight.bold)),
      legend:
          Legend(isVisible: true, overflowMode: LegendItemOverflowMode.wrap),
      series: _getDefaultDoughnutSeries(),
      tooltipBehavior: TooltipBehavior(enable: true),
    );
  }

  /// Returns the doughnut series which need to be render.
  List<DoughnutSeries<ChartSampleData, String>> _getDefaultDoughnutSeries() {
    final List<ChartSampleData> chartData = <ChartSampleData>[
      ChartSampleData(x: 'Men', y: 55, text: '55%'),
      ChartSampleData(x: 'Woman', y: 31, text: '45%'),
    ];
    return <DoughnutSeries<ChartSampleData, String>>[
      DoughnutSeries<ChartSampleData, String>(
          radius: '80%',
          explode: true,
          explodeOffset: '10%',
          dataSource: chartData,
          xValueMapper: (ChartSampleData data, _) => data.x,
          yValueMapper: (ChartSampleData data, _) => data.y,
          dataLabelMapper: (ChartSampleData data, _) => data.text,
          dataLabelSettings: DataLabelSettings(isVisible: true))
    ];
  }
}```

**Comment by Dharanipalani** [comment]
Hi @jordandevogelaere,

Thanks for the interest in our Flutter charts. We have analyzed your query and the given code and unfortunately, we are unable to replicate the reported issue at our end. We have checked in the android, and the web in both debug and release modes. If you share more information like the platform you are facing the issue, and some replicating procure with the sample will help us to analyze and provide the solution to you sooner.

Thanks,
Dharanitharan. P
